packages/ariakit/src/checkbox/__examples__/checkbox-group/test.tsx

```diff
@@ -0,0 +1,109 @@
+import { click, getByLabelText, press, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render checkbox", async () => {
+  const { container } = render(<Example />);
+  expect(container).toMatchInlineSnapshot(`
+    <div>
+      <div
+        aria-labelledby="r:0"
+        class="group"
+        role="group"
+      >
+        <div
+          aria-hidden="true"
+          class="group-label"
+          id="r:0"
+        >
+          Your favorite fruits
+        </div>
+        <br />
+        <label
+          class="label"
+        >
+          <input
+            aria-checked="false"
+            class="checkbox"
+            data-command=""
+            type="checkbox"
+            value="apple"
+          />
+           Apple
+        </label>
+        <label
+          class="label"
+        >
+          <input
+            aria-checked="false"
+            class="checkbox"
+            data-command=""
+            type="checkbox"
+            value="orange"
+          />
+           Orange
+        </label>
+        <label
+          class="label"
+        >
+          <input
+            aria-checked="false"
+            class="checkbox"
+            data-command=""
+            type="checkbox"
+            value="mango"
+          />
+           Mango
+        </label>
+      </div>
+    </div>
+  `);
+});
+
```

Comment:
We're trying to have as few snapshot tests as possible. Since we already have snapshots for group and checkbox, I'd say we don't need this here.
```suggestion

```